src/app/shared/entry/recent-events.pipe.ts

```diff
@@ -25,27 +24,15 @@ export class RecentEventsPipe implements PipeTransform {
 
     switch (type) {
       case 'displayName': {
-        if (event.workflow) {
-          return this.entryToDisplayNamePipe.transform(event.workflow);
-        } else if (event.tool) {
-          return this.entryToDisplayNamePipe.transform(event.tool);
-        } else if (event.apptool) {
-          return this.entryToDisplayNamePipe.transform(event.apptool);
-        }
-        break;
+        const entry = this.getEntry(event);
+        return entry && this.entryToDisplayNamePipe.transform(entry);
       }
       case 'entryLink': {
-        if (event.workflow) {
-          return '/workflows/' + event.workflow.full_workflow_path;
-        } else if (event.tool) {
-          return '/containers/' + event.tool.tool_path;
-        } else if (event.apptool) {
-          return '/containers/' + event.apptool.full_workflow_path;
-        }
-        break;
+        const entry = this.getEntry(event);
+        return entry && '/' + entry.entryTypeMetadata.sitePath + '/' + this.getPath(entry);
```

Comment:
Our autoformatter removed the parentheses around the subexpression on the right side of the `&&`, it was better with them.

Reply:
Maybe I should change the right subexpression to a template literal.